packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -119,9 +111,9 @@ export default class VTable {
                     }
                 });
             });
-            if (this.formatInfo) {
-                this.applyFormat(this.formatInfo);
-                saveTableInfo(this.table, this.formatInfo);
+            const formatInfo = this.getTableFormatInfo();

```

Comment:
Please see the design, we should still get formatTable in constructor, then never read it from table again.

When writeBack(), call applyTableFormat() if tableFormat exists.

Reply:
<img width="463" alt="image" src="https://user-images.githubusercontent.com/23065085/152891276-ff3e89d3-8d94-4cd9-b3cd-8dbb273dd126.png">

Reply:
We should save the format before we create the table, otherwise formatInfo will be undefined, then we cannot not change the state of the table.

Reply:
That is right. And that is why you need to read it in constructor, and save into this.tableFormat.